src/main/java/com/example/cookbook/recipesCategory/recipies/Recipe.java

```diff
@@ -0,0 +1,98 @@
+package com.example.cookbook.recipesCategory.recipies;
+
+import com.example.cookbook.recipesCategory.RecipeCategory;
+import com.example.cookbook.recipesCategory.recipies.liker.Liker;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+public class Recipe {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String name;
+    private String recipeUrl;
+
+    private Integer prepareTime;
+
+    @ManyToOne
+    private RecipeCategory recipeCategory;
+
+    @OneToMany(mappedBy = "recipe")
+    private List<RecipeDescription> recipeDescriptionList;
+
+    @OneToMany(mappedBy = "recipe")
+    private List<Liker> likers;
```

Comment:
Wydzielenie osobnej klasy dla kategorii jest jak najbardziej ok, natomiast dwie pozostałe encje, tzn. RecipeDescription i Liker już utworzyłeś niepotrzebnie.
Pola z RecipeDescription dodałbym bezpośrednio do Recipe, a do liczby polubień mogłeś użyć zwykłego Integera.
To generalnie bardzo upraszcza całą aplikację i pozwala się pozbyć sporej części kodu